src/main/java/baritone/process/BackfillProcess.java

```diff
@@ -124,6 +124,7 @@ public void onLostControl() {
         if (blocksToReplace != null && !blocksToReplace.isEmpty()) {
             blocksToReplace.clear();
         }
+        super.onLostControl();
```

Comment:
this is a reasonable way to do this. have you checked if cancel will cancel one or all?